projects/plugins/jetpack/_inc/blogging-prompts.php

```diff
@@ -258,11 +258,104 @@ function jetpack_is_new_post_screen() {
 	return false;
 }
 
+/**
+ * Determines if the user selected a "blogging" intent or goal when creating the site.
+ *
+ * @return bool
+ */
+function _jetpack_has_selected_blogging_goal_or_intent() {
+	$intent = get_option( 'site_intent', '' );
+	if ( 'write' === $intent || 'design-first' === $intent || 'start-writing' === $intent ) {
```

Comment:
Nitpick: what would you say for using `in_array` here as well, to be consistent with what's below, and maybe to increase readability a bit?

```suggestion
	if ( in_array( $intent, array( 'write', 'design-first', 'start-writing' ), true ) ) {
```